docusaurus/docs/ui-extensions/register-an-extension.md

```diff
@@ -0,0 +1,128 @@
+---
+id: register-an-extension
+title: Register an extension from your app plugin
+sidebar_label: Register an extension
+sidebar_position: 1
+description: Learn how to register an extension from your app plugin.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - links
+  - extensions
+  - app plugins
+---
+
+## Before you begin
+
+Be sure your plugin meets the following requirements before proceeding:
+
+- It must be an app plugin.
+- It must be preloaded by setting the [preload property](https://grafana.com/docs/grafana/latest/developers/plugins/metadata/#pluginjson) to `true` in the `plugin.json`.
+- It must be installed and *enabled*.
+
+## How to add an extension to Grafana core/plugins from your app plugin
+
+1. Open the `module.ts(x)` file in your plugin where your `AppPlugin` should be exposed.
```

Comment:
Is this step required? 

" in your plugin where your `AppPlugin` should be exposed." feels like overkill. You'd only have one module.ts right?

Reply:
Agree 👍🏻